test/jasmine/specs/global-object-css-spec.js

```diff
@@ -12,6 +12,24 @@ describe("Test getting JSON from <body/> CSS", function () {
         );
     });
 
+    it("should get expected strings from global element", function () {
```

Comment:
Do you think this test is needed?
Unfortunately, 2 tests are still failing after the changes in this PR:
```
X shouldn convert CSS to JSON from provided CSS property in CSS generated elementSass string value for which was provided with property name
     Expected undefined to equal 'colorMap'. (1)
   X shouldn convert CSS to JSON from provided CSS property in CSS generated elementSass complex value for which was provided with property name
     Expected undefined to equal Object({ colorHexShort: '#f00', colorHex: '#ff0000', colorRgba: 'rgba(255,0,0,0.5)' }). (1)

```